codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HeaderField.java

```diff
@@ -31,7 +31,6 @@
  */
 package io.netty.handler.codec.http2.internal.hpack;
 
-import static io.netty.util.CharsetUtil.ISO_8859_1;
 import static io.netty.util.internal.ObjectUtil.checkNotNull;
 
 class HeaderField implements Comparable<HeaderField> {
```

Comment:
followup pr: does this class need to be comparable? We go through the effort to have an equals that "doesn't leak timing info" ... but the compare function doesn't appear to do the same.